packages/shadcn/package.json

```
@@ -0,0 +1,113 @@
+{
+  "name": "@rjsf/shadcn",
+  "version": "6.0.0-alpha.0",
+  "main": "dist/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "description": "shadcn theme, fields and widgets for react-jsonschema-form",
+  "files": [
+    "dist",
+    "lib",
+    "src"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rjsf-team/react-jsonschema-form.git"
+  },
+  "scripts": {
+    "build:css": "node build-css.cjs",
+    "build:ts": "tsc -b",
```

To improve ESM support you will need to change this to:
```suggestion
        "build:ts": "tsc -b tsconfig.build.json && tsc-alias -p tsconfig.build.json",
```
And add a `tsconfig.build.json` that contains:
```
{
  "extends": "../../tsconfig.build.json",
  "compilerOptions": {
    "outDir": "./lib"
  },
  "files": [],
  "references": [
    {
      "path": "./src"
    }
  ],
  "tsc-alias": {
    "resolveFullPaths": true,
    "verbose": true,
  }
}
```